warehouse/models/mart/gtfs_schedule_latest/_gtfs_schedule_latest.yml

```diff
@@ -640,8 +735,6 @@ models:
           `25:40:00`, which are allowed in GTFS.
 
           See: https://gtfs.org/schedule/reference/#field-types for how GTFS defines its "Time" type.
-        meta:
```

Comment:
If we flip these switches in this PR, these columns will get published without documentation on Monday, right? Wondering if we shouldn't wait to flip these until we add the keys for the new tables?

Reply:
Resolved in [845ee88](https://github.com/cal-itp/data-infra/pull/2667/commits/845ee886452e88adf1e3544c182a50e6273afee7), that was just an oversight in the previous commit when other publish.ignore calls were restored.